### PR TITLE
HOCS-1220 - Work stack changes for WCS

### PR DESCRIPTION
**Pull request by hubert-raich**

### CODE REVIEW

File: src/main/resources/db/postgresql/V1_19__CREATE_WORKSTACK_COLUMN.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
+DROP TABLE IF EXISTS workstack_column;
```
Comment (lewismuirhead) [inline src/main/resources/db/postgresql/V1_19__CREATE_WORKSTACK_COLUMN.sql]: should this file be included?

File: src/main/resources/db/postgresql/V1_18__CREATE_TABLE_FILTER_FIELD.sql [hunk 2]
```diff
@@ -0,0 +1,26 @@
+DROP TABLE IF EXISTS search_field;
```
Comment (lewismuirhead) [inline src/main/resources/db/postgresql/V1_18__CREATE_TABLE_FILTER_FIELD.sql]: should this file be included?

File: src/main/resources/db/postgresql/V1_17__CREATE_TABLE_COUNTRY.sql [hunk 3]
```diff
@@ -0,0 +1,14 @@
+DROP TABLE IF EXISTS country;
```
Comment (lewismuirhead) [inline src/main/resources/db/postgresql/V1_17__CREATE_TABLE_COUNTRY.sql]: should this file be included?

File: src/main/resources/db/postgresql/V1_16__CREATE_TABLE_SYSTEM_CONFIGURATION.sql [hunk 4]
```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS system_configuration;
```
Comment (lewismuirhead) [inline src/main/resources/db/postgresql/V1_16__CREATE_TABLE_SYSTEM_CONFIGURATION.sql]: should this file be included?

Comment (hubert-raich) [reply src/main/resources/db/postgresql/V1_16__CREATE_TABLE_SYSTEM_CONFIGURATION.sql]: yes, as discussed it's for flyway consistency when we merge wcs branch back to cs

File: src/main/resources/db/postgresql/V1_17__CREATE_TABLE_COUNTRY.sql (hunk 3 above)
Comment (hubert-raich) [reply src/main/resources/db/postgresql/V1_17__CREATE_TABLE_COUNTRY.sql]: yes, as discussed it's for flyway consistency when we merge wcs branch back to cs

File: src/main/resources/db/postgresql/V1_18__CREATE_TABLE_FILTER_FIELD.sql (hunk 2 above)
Comment (hubert-raich) [reply src/main/resources/db/postgresql/V1_18__CREATE_TABLE_FILTER_FIELD.sql]: yes, as discussed it's for flyway consistency when we merge wcs branch back to cs

File: src/main/resources/db/postgresql/V1_19__CREATE_WORKSTACK_COLUMN.sql (hunk 1 above)
Comment (hubert-raich) [reply src/main/resources/db/postgresql/V1_19__CREATE_WORKSTACK_COLUMN.sql]: yes, as discussed it's for flyway consistency when we merge wcs branch back to cs

**Pull request merged**